specifyweb/frontend/js_src/lib/components/Attachments/Viewer.tsx

```diff
@@ -94,11 +94,15 @@ export function AttachmentViewer({
   const table = f.maybe(serialized.tableID ?? undefined, getAttachmentTable);
   return (
     <div className="flex h-full gap-8">
-      <div className="flex min-h-[30vw] w-full min-w-[30vh] flex-1 items-center">
+      <div className="flex max-h-screen min-h-[30vw] w-full min-w-[30vh] flex-1 items-center">
```

Comment:
the image sizing behaves nicely now, but the image is still not horizontally centered

Reply:
also a separate improvement:
`min-h-[30vw] min-w-[30vh]` is fine on a regular screen, but if you zome out a lot (to simulate a large screen), 30% of the screen is way too much.
Can you try out some rem-based value instead?
From quick testing, `min-w-[theme(spacing.60)] min-h-[theme(spacing.60)]` seems to work, but you should try and see what works best. 
Example way to test this: upload a pdf and shrink dialog size and make sure the preview does not shrink beyond so much that it's not useful anymore (could also try this when the attachment plugin is on the form)